mne_bids/commands/mne_bids_mark_bad_channels.py

```diff
@@ -0,0 +1,89 @@
+"""Mark channels in an existing BIDS dataset as "bad".
+
+example usage:
+$ mne_bids mark_bad_channels --ch_name="MEG 0112" --description="noisy" \
+                             --ch_name="MEG 0131" --description="flat" \
+                             --subject_id=01 --task=experiment --session=test \
+                             --bids_root=bids_root --overwrite
+
+"""
+# Authors: Richard Höchenberger <richard.hoechenberger@gmail.com>
+#
+# License: BSD (3-clause)
+
+import mne_bids
+from mne_bids import BIDSPath, mark_bad_channels
+
+
+def run():
+    """Run the mark_bad_channels command."""
+    from mne.commands.utils import get_optparser
+
+    parser = get_optparser(__file__, usage="usage: %prog options args",
+                           prog_prefix='mne_bids',
+                           version=mne_bids.__version__)
+
+    parser.add_option('--ch_name', dest='ch_names', action='append',
+                      default=[],
+                      help='The names of the channels, separated by commas')
+    parser.add_option('--description', dest='descriptions', action='append',
+                      default=[],
+                      help='Descriptions as to why the channels are bad')
```

Comment:
Say also it can be specified multiple times and that it should match the number of bad channels